runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsConfig.java

```diff
@@ -148,6 +148,61 @@ default Color getNonClanMemberColor()
 
 	@ConfigItem(
 		position = 10,
+		keyName = "drawClanCallerNames",
+		name = "Draw Clan caller names",
+		description = "Configures whether or not clan caller names should be drawn"
+	)
+	default boolean drawClanCallerNames()
```

Comment:
This could probably be just changed to "highlighted" player and player highlighting to be more generic